src/write_gcm.c

```diff
@@ -2836,6 +2848,10 @@ static int wg_json_CreateCollectdTimeseriesRequest(_Bool pretty,
   wg_json_string(jc, "name");
   wg_json_string(jc, name);
 
+  // Request partial errors from the server, so we can log them.
+  wg_json_string(jc, "return_partial_errors");
```

Comment:
That's the Logging API. The reason to diverge is that in the Logging API it changes the server-side behavior on whether to fail the call on any error. In CreateCollectdTimeSeries we always use partial failures and we don't see a reason to change that. The reason I'm introducing this behind a flag is that it will increase the response size for an unknown number of requests, and I don't see a good reason to do that.